docs/migration_guides/opflow_migration.rst

```diff
@@ -0,0 +1,1156 @@
+    or by supplying a method instance of :class:`~qiskit.opflow.evolutions.Trotter`, :class:`~qiskit.opflow.evolutions.Suzuki` or :class:`~qiskit.opflow.evolutions.QDrift`).
+
+    The different trotterization methods that extend :class:`~qiskit.opflow.evolutions.TrotterizationBase` were migrated to :mod:`~qiskit.synthesis`,
+    and now extend the :class:`~qiskit.synthesis.evolution.ProductFormula` base class. They no longer contain a ``.convert()`` method for
+    standalone use, but now are designed to be plugged into the :class:`~qiskit.synthesis.PauliEvolutionGate` and called via ``.synthesize()``.
+    In this context, the job of the :class:`~qiskit.opflow.evolutions.PauliTrotterEvolution` class can now be handled directly by the algorithms
+    (for example, :class:`~qiskit.algorithms.time_evolvers.TrotterQRTE`\).
+
+    In a similar manner, the :class:`~qiskit.opflow.evolutions.MatrixEvolution` class performs evolution by classical matrix exponentiation,
+    constructing a circuit with :class:`~UnitaryGate`\s or :class:`~HamiltonianGate`\s containing the exponentiation of the operator.
+    This class is no longer necessary, as the :class:`~HamiltonianGate`\s can be directly handled by the algorithms.
+
+Trotterizations
+~~~~~~~~~~~~~~~
+*Back to* `Index`_
+
+.. list-table::
+   :header-rows: 1
+
+   * - Opflow
+     - Alternative
+
+   * - :class:`~qiskit.opflow.evolutions.TrotterizationFactory`
+     - No direct replacement. This class was used to create instances of one of the classes listed below.
+
+   * - :class:`~qiskit.opflow.evolutions.Trotter`
+     - :class:`~synthesis.SuzukiTrotter` or :class:`~synthesis.LieTrotter`
+
+   * - :class:`~qiskit.opflow.evolutions.Suzuki`
+     - :class:`~synthesis.SuzukiTrotter`
+
+   * - :class:`~qiskit.opflow.evolutions.QDrift`
+     - :class:`~synthesis.QDrift`
+
+Other Evolution Classes
+~~~~~~~~~~~~~~~~~~~~~~~~
+*Back to* `Index`_
+
+.. list-table::
+   :header-rows: 1
+
+   * - Opflow
+     - Alternative
+
+   * - :class:`~qiskit.opflow.evolutions.EvolutionFactory`
+     - No direct replacement. This class was used to create instances of one of the classes listed below.
+
+   * - :class:`~qiskit.opflow.evolutions.EvolvedOp`
+     - No direct replacement. The workflow no longer requires a specific operator for evolutions.
+
+   * - :class:`~qiskit.opflow.evolutions.MatrixEvolution`
+     - :class:`~HamiltonianGate`
+
+   * - :class:`~qiskit.opflow.evolutions.PauliTrotterEvolution`
+     - :class:`~PauliEvolutionGate`
+
+
+Example 1: Trotter evolution
+############################
+
+.. raw:: html
+
+    <details>
+    <summary><a><b>Opflow</b></a></summary>
+
+.. code-block:: python
+
+    from qiskit.opflow import Trotter, PauliTrotterEvolution, PauliSumOp
+
+    hamiltonian = PauliSumOp.from_list([('X', 1), ('Z',1)])
+    evolution = PauliTrotterEvolution(trotter_mode=Trotter(), reps=1)
+    evol_result = evolution.convert(hamiltonian.exp_i())
```

Comment:
I think it would be cool to show how the `time` parameter is handled:
```
(time * hamiltonian).exp_i()
```
vs
```
PauliEvolutionGate(hamiltonian, time=time)
```
😄